README.md

```diff
@@ -131,6 +131,8 @@ If you're transpiling (and/or using the `jsnext:main`):
 
 ```javascript
 import glamorous, {ThemeProvider} from 'glamorous'
+// you can also import specific components
+import {Div, H2} from 'glamorous'
```

Comment:
Could you include a line about the special case for JavaScript built-ins like `Object`? Something like:

> // Tags with the same name as built-in JavaScript objects are importable with a `Tag` suffix, IE: `ObjectTag`.